src/verifast1.ml

```diff
@@ -397,6 +397,7 @@ module VerifyProgram1(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
   let ancestry_symbol = mk_symbol "ancestry" [ctxt#type_int] ctxt#type_inductive Uninterp
   let ancester_at_symbol = mk_symbol "ancester_at" [ctxt#type_int; ctxt#type_int] ctxt#type_int Uninterp
   let get_class_symbol = mk_symbol "getClass" [ctxt#type_int] ctxt#type_int Uninterp
+  let cxx_typeid_symbol = mk_symbol "typeid" [ctxt#type_int] ctxt#type_int Uninterp

```

Comment:
Since the provenance commits, the provertype of a pointer is now `inductive`, not `int`.

Reply:
Changed the domain and range to `inductive` type